javascript/operators/import.json

```
@@ -65,6 +65,65 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options": {
+          "__compat": {
+            "description": "The second options parameter",
```

The spec seems to call it `opt` for "options" so this sounds good to me -- let's call it the "options" parameter to adhere to the existing structure of these parameters throughout BCD!

(Note: we'll want to make sure to call it as such on MDN as well.)

```suggestion
        },
        "options_parameter": {
          "__compat": {
            "description": "The <code>options</code> parameter",
```